demos/README.md

```diff
@@ -32,7 +32,6 @@ ovms_demo_real_time_stream_analysis
 ovms_demo_using_paddlepaddle_model
 ovms_demo_bert
 ovms_demo_universal-sentence-encoder
-ovms_demo_speech_recognition
```

Comment:
Why is it removed? is it 404?

Reply:
I have removed this reference due to the fact that there is no such demo. It is available in the 2023.2 version:

https://github.com/openvinotoolkit/model_server/tree/releases/2023/2/demos/speech_recognition_with_kaldi_model/python